### ISSUE TITLE
3dtiles not show in  3DTILES_implicit_tiling extension  with root  transform property

### ISSUE THREAD

**Issue by lch710097** [CesiumGS/cesium-unreal#766]
3dtiles not show in  3DTILES_implicit_tiling extension  with root  transform property.
while change  Cesium3DTilesSelection\src\TilesetLoadTileFromJson.cpp,   childTile.setTransform(tile.getTransform());only show first level data,and no details data.
![changesource](https://user-images.githubusercontent.com/21329096/155688468-d6597fc7-b713-4ae1-84f9-2891ba2a68ce.png)
![root](https://user-images.githubusercontent.com/21329096/155688477-7167c1dd-ca4a-4b66-b744-1e414b095a63.png)
![tileing](https://user-images.githubusercontent.com/21329096/155688481-9e86fce4-0a27-41b9-8e47-e0f44b4831bb.png)

**Comment by nithinp7** [comment]
Thanks for reporting this @lch710097, I will apply your fix today. I will look into if there is a second bug as well.

**Comment by nithinp7** [comment]
Hey @lch710097, can you check if this PR fixes your issues: https://github.com/CesiumGS/cesium-native/pull/453?

**Comment by lch710097** [comment]
![data](https://user-images.githubusercontent.com/21329096/156170178-148a6fa5-de8f-428b-9d97-6e67f5df0228.png)
![t1](https://user-images.githubusercontent.com/21329096/156170189-e0f0d3c6-1b8d-4281-9190-58396bb44627.png)
![t2](https://user-images.githubusercontent.com/21329096/156170191-f9891e65-37fa-4c5a-bd02-67b6d558beac.png)
![t3](https://user-images.githubusercontent.com/21329096/156170194-dd57e2d4-9e67-4414-8a92-70f6e71d3f8e.png)
![tile](https://user-images.githubusercontent.com/21329096/156170199-79de1b76-5d94-41e2-b8dc-3d34144d6730.png)
![web](https://user-images.githubusercontent.com/21329096/156170201-18922951-a28a-4850-9063-d79eccda54c5.png)
hei, the issues is not fixed,after change the code, it's only show 0 and 1 level data ,no more details data.
please see the attach  data.png is the result show in ue,tile.png is the request url ,and t1.png t2.png t3.png is the code to change .
web.png is tile data show in cesium web

**Comment by lch710097** [comment]
[https://drive.google.com/file/d/1tPwAWhOYPXrnzutnT7ICOrOF7VqhpiPQ/view](test tile data)

**Comment by lch710097** [comment]
https://drive.google.com/file/d/1tPwAWhOYPXrnzutnT7ICOrOF7VqhpiPQ/view is test tile data

**Comment by nithinp7** [comment]
I don't think we confirmed a fix to the second issue, so I'll reopen it for now.

**Comment by j9liu** [comment]
This issue is pretty old, and 3D Tiles 1.1 supports implicit tiling directly without the `3DTILES_implicit_tiling` extension. I'll close this issue unless this is still a problem for users.
